src/Titon/Context/Definition/DefinitionInterface.hh

```diff
@@ -0,0 +1,36 @@
+<?hh // strict
+/**
+ * @copyright   2010-2015, The Titon Project
+ * @license     http://opensource.org/licenses/bsd-license.php
+ * @link        http://titon.io
+ */
+
+namespace Titon\Context\Definition;
+
+/**
+ * DefinitionInterface for depository definitions to follow
+ *
+ * @package Titon\Context\Definition
+ */
+interface DefinitionInterface
```

Comment:
Really, I can just move ```create``` into AbstractDefinition as an abstract method. That sound reasonable?